src/crimson/os/seastore/cache.h

```diff
@@ -1328,6 +1403,69 @@ class Cache {
     );
   }
 
+  template <typename T>
+  get_extent_ret<T> read_extent(
+    TCachedExtentRef<T>&& extent,
+    extent_len_t offset,
+    extent_len_t length
+  ) {
+    if (extent->is_ptr()) {
+      return read_extent(std::move(extent));
+    }
+    assert(extent->state == CachedExtent::extent_state_t::CLEAN_PENDING);
+    extent->set_io_wait();
+    auto old_length = extent->get_valid_length();
+    return seastar::do_with(
+      extent->read_buffer(offset, length),
+      [extent, this](auto &read_regions) {
+        return read_ertr::parallel_for_each(
+          read_regions,
+          [extent, this](auto &read_region) {
+            extent_len_t start = p2align(read_region.first,
+              epm.get_block_size());
+            extent_len_t end = p2roundup(
+              read_region.first + read_region.second,
+              epm.get_block_size());
+            extent_len_t aligned_length = end - start;
+            return seastar::do_with(
+              alloc_cache_buf(aligned_length),
+              [extent, this, start, aligned_length](auto &ptr){
+                return epm.read(
```

Comment:
Let's reduce the unnecessary indention: make sure that the nested logic only has 2 extra spaces, not 4.

For example: https://github.com/ceph/ceph/blob/6e67bf80aa03c99a6df96e3cf7ea6416bbfc8054/src/crimson/os/seastore/async_cleaner.cc#L1068-L1079